src/arkode/arkode_butcher.c

```diff
@@ -392,25 +392,31 @@ void ARKodeButcherTable_Write(ARKodeButcherTable B, FILE* outfile)
     fprintf(outfile, "      ");
     for (j = 0; j < B->stages; j++)
     {
-      fprintf(outfile, "%" RSYM "  ", B->A[i][j]);
+      fprintf(outfile, SUN_FORMAT_E "  ", B->A[i][j]);
```

Comment:
I'm a bit confused about the logic between when some `RSYM` were replaced by `SUN_FORMAT_G` and others by `SUN_FORMAT_E`, particularly here since I do not think of Butcher table coefficients as being particularly large or small, whereas in `arkode_erkstep.c` the value of `ark_mem->tcur` (that could have magnitude far different from 1) uses `SUN_FORMAT_G`.

Reply:
The general convention is `SUN_FORMAT_E` for vectors, matrices, tables, etc where having the numbers consistently aligned is ideal. Everything else uses `SUN_FORMAT_G` which is probably the most reader-friendly format.

Reply:
This convention sounds good to me. It should be added to the coding [conventions list](https://sundials.readthedocs.io/en/develop/developers/source_code/Rules.html) in the developer guide